supportedBackends/local/src/test/scala/cromwell/backend/impl/local/LocalJobExecutionActorSpec.scala

```diff
@@ -19,11 +19,10 @@ import wdl4s.values._
 
 import scala.concurrent.duration._
 
-class LocalJobExecutionActorSpec extends FlatSpec with BackendTestkitSpec with MockitoSugar with TestFileUtil with TableDrivenPropertyChecks {
+class LocalJobExecutionActorSpec extends FlatSpec with BackendTestkitSpec with MockitoSugar with TestFileUtil
+  with TableDrivenPropertyChecks with HasLocalBackendConfig {
 
-  val globalConfig = ConfigFactory.load()
-  val backendConfig = globalConfig.getConfig("backend.providers.Local.config")
-  val defaultBackendConfigDescriptor = new BackendConfigurationDescriptor(backendConfig, globalConfig)
+  val defaultBackendConfigDescriptor = new BackendConfigurationDescriptor(localBackendConfig, globalConfig = null)
```

Comment:
Interesting choice on swapping in the `globalConfig = null`. I'm going to pretend this is just like a "mock" and look the other way.